common/config/rush/npm-shrinkwrap.json

```diff
@@ -3125,14 +3145,14 @@
       "resolved": "https://registry.npmjs.org/js-tokens/-/js-tokens-3.0.2.tgz"
     },
     "js-yaml": {
-      "version": "3.9.1",
-      "from": "js-yaml@>=3.0.0 <4.0.0",
-      "resolved": "https://registry.npmjs.org/js-yaml/-/js-yaml-3.9.1.tgz",
+      "version": "3.8.4",
```

Comment:
Erm... no, good catch.

---
In reply to: [134340149](https://github.com/Microsoft/web-build-tools/pull/325#discussion_r134340149) [](ancestors = 134340149)